### DISCUSSION TITLE
Having trouble installing xournalpp

### DISCUSSION THREAD

**Discussion by Al-Batal** [xournalpp/xournalpp#5714] [Q&A]
I'm trying to install xournalpp in windows, but I'm having troubles because my virus scanner keeps blocking the installation. I get error saying xournalpp is infected with FileRepMalware.

**Comment by GithubAnon0000** [comment]
Can't help with installing on windows but according to virustotal there is not a single thing that even might be related to malware inside the [windows binary](https://github.com/xournalpp/xournalpp/releases/download/v1.2.3/xournalpp-1.2.3-windows.zip) (or any other xournalpp file) from github.

You should search on the internet on how to install an .exe file if your specfic antivirus app tries to prevent it.
